monitoring/README.md

```diff
@@ -153,6 +157,7 @@ a) Port-forward Grafana to localhost, by using the kubectl command:
 ```bash
 kubectl --context kind-kubeflex port-forward -n ks-monitoring svc/grafana 3000:80
 ```
+For OpenShift clusters get the OpenShift route for the Grafana service in the `ks-monitoring` namespace.
```

Comment:
I do not understand this sentence. What exactly is it telling me to get (an API object, a URL, ...?), and what am I supposed to do with it?